ccd-definition/AuthorisationCaseState.json

```diff
@@ -145,5 +145,26 @@
     "CaseStateID": "PREPARE_FOR_HEARING",
     "UserRole": "caseworker-publiclaw-systemupdate",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseStateID": "Submitted",
+    "UserRole": "caseworker-publiclaw-bulkscan",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseStateID": "Gatekeeping",
+    "UserRole": "caseworker-publiclaw-bulkscan",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseStateID": "PREPARE_FOR_HEARING",
+    "UserRole": "caseworker-publiclaw-bulkscan",
+    "CRUD": "CRU"
```

Comment:
happy to allow from any state, I don't think there would be any cases where this would happen however. HMCTS staff don't have access to a case until submitted state so would never be able to handle exception records etc